src/brsTypes/components/Font.ts

```diff
@@ -0,0 +1,17 @@
+import { RoSGNode, FieldModel } from "./RoSGNode";
+import { AAMember } from "./RoAssociativeArray";
+
+export class Font extends RoSGNode {
+    readonly defaultFields: FieldModel[] = [
+        { name: "uri", type: "uri" },
```

Comment:
[The docs](https://developer.roku.com/en-gb/docs/references/scenegraph/typographic-nodes/font.md) technically say `uri` is the type for Font, but it's pretty clear they represent it as a `BrsString`. Happy to make this `string` instead of `uri` if that seems more appropriate.

Reply:
`uri` appears to be a valid type for scenegraph components, but it appears to be an alias for `string` — I don't see any URI validation after creating a `Font` node in RBI!

Reply:
Yeah, I saw that as well -- plus, I noticed that I could perform all of the normal `string` operations on it.